### ISSUE TITLE
When leaving path variable blank, the variable name with a prepended colon gets sent as the variable instead of blank or null

### ISSUE THREAD

**Issue by dave4jr** [a85/POSTMan-Chrome-Extension#12630]
### Is there an existing issue for this?

- [X] I have searched the tracker for existing similar issues and I know that duplicates will be closed

### Describe the Issue

When sending a basic GET request, `https://domain.com/users/:id`, if the value for `id` path variable is left empty, the actual request that gets sent is https://domain.com/users/:id, with :id as a string as the variable, instead of https://domain.com/users/null or https://domain.com/users/. So in other words, if a path variable named var is left empty, the string :var is sent in it's place instead of sending null or empty string. I looked all over the postman documentation and previous issues and couldn't find an answer to this.

### Steps To Reproduce

1. Open a new GET request tab with a variable path variable named id
2. Clear the id path value
3. Send request and view request 

### Screenshots or Videos

<img width="754" alt="Screenshot 2024-02-07 at 6 07 34 PM" src="https://github.com/postmanlabs/postman-app-support/assets/7722220/af420004-95a2-441b-b1a7-03fbde2af62b">
<img width="1487" alt="Screenshot 2024-02-07 at 6 07 10 PM" src="https://github.com/postmanlabs/postman-app-support/assets/7722220/8cc85ec6-c90c-4fd3-a012-7185185499eb">

### Operating System

macOS

### Postman Version

10.22.13

### Postman Platform

Postman App

### User Account Type

Signed In User

### Additional Context?

Basically I want the variable to come through as either null or blank so I can tell the user or whoever sent the request that they must provide that variable, or be able to set a default path variable if none is provided. I'm almost positive this is how it used to work but could be wrong somehow. Thank you!

**Comment by dave4jr** [comment]
Any update on this?

**Comment by dave4jr** [comment]
Can anyone give me their thoughts on this? Is this something that you guys are planning on fixing at some point? Any response would be awesome, thank you!

**Comment by dave4jr** [comment]
@ronork Any plans to fix this going forward?

**Comment by ronork** [comment]
@dave4jr Thanks for bringing this up. While I understand the issue,  reason for keeping the path variables as strings is to maintain flexibility. This allows users to potentially send ':id' as a literal value if needed. If you want to send a null value, you can [define a variable](https://learning.postman.com/docs/sending-requests/variables/variables/#defining-variables) and explicitly set it to null.

**Comment by RokoStar** [comment]
@ronork how can I set the variable as a null? can't find it anywhere in docs

**Comment by risinek** [comment]
based on the hint by @ronork, I have created a global variable with no value, i.e. null. So when I want to omit path variable in URL, I just use this "null" variable as a value.

![Screenshot 2024-04-18 at 10 13 27](https://github.com/postmanlabs/postman-app-support/assets/12913557/7132681b-d65a-449a-b3eb-c42f6e5283f7)
